### PR TITLE
feat(oui-navbar): allow to have custom content in menu header

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change
* Add possibility to add custom content with header title in menus 

⚠️ BREAKING CHANGE

### CODE REVIEW

File: packages/oui-navbar/README.md [hunk 1]
```diff
@@ -46,9 +46,8 @@
             icon-class="{{asideLink.iconClass}}"
             on-click="asideLink.onClick"
             ng-repeat="asideLink in $ctrl.asideLinks track by $index"
-            ng-class="asideLink.class"
-            ng-switch="asideLink.name">
-            <oui-navbar-notification ng-switch-when="notifications"
+            ng-class="asideLink.class">
+            <oui-navbar-notification ng-if="asideLink.name === 'notifications'"
```
Comment (FredericEspiau) [inline packages/oui-navbar/README.md]: Why did you remove the `switch` ?

File: packages/oui-navbar/src/menu/navbar-menu.html [hunk 2]
```diff
@@ -96,6 +99,9 @@ <h3 class="oui-navbar-menu__title" ng-bind="::$ctrl.headerTitle" title="{{::$ctr
             align="{{::$ctrl.align}}"
             fixed="::$ctrl.fixed"
             back-button>
+            <oui-navbar-menu-header>
+                <div ng-transclude="header"></div>
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/oui-navbar/src/menu/navbar-menu.html]: ?

[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/oui-navbar/src/menu/navbar-menu.html]: Same for the possible empty `div` if `header` is empty

File: packages/oui-navbar/README.md (hunk 1 above)
Comment (marie-j) [reply packages/oui-navbar/README.md]: Multiple directive asking for transclusion

File: packages/oui-navbar/src/index.spec.js [hunk 3]
```diff
@@ -441,6 +441,37 @@ describe("ouiNavbar", () => {
                 menu = component.find("oui-navbar-menu");
                 expect(menu.hasClass(navbarMenuFixedClass)).toBeTruthy();
             });
+
+            it("should be have custom content", () => {
```
Comment (antleblanc) [inline packages/oui-navbar/src/index.spec.js]:
```suggestion
            it("should have custom content", () => {
```
?
